### PR TITLE
Fix compilation on Node v10.11

### PR DESCRIPTION
**Pull request by kevinburke**
Attempting to run "npm install" on a new-ish Mac with Node v10.11
fails with a number of node-gyp compilation failures. Installing and
using the latest versions of node-sass, nan, and fsevents resolves
this failure, as each of those packages resolves the issues that led
to problems.

Update lodash to v4.17.11 to fix a security warning present in the
Travis CI configuration.

I am nervous about the number of changes to package-lock.json and I am
not sure how to mitigate that.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,10 +47,12 @@
     "eslint-config-webpack": "^1.2.1",
     "eslint-plugin-import": "^2.2.0",
     "extract-text-webpack-plugin": "^3.0.0",
+    "fsevents": "^1.2.4",
     "jest": "^19.0.2",
     "lint-staged": "^3.4.0",
-    "lodash": "^4.17.4",
-    "node-sass": "^4.5.3",
+    "lodash": "^4.17.11",
+    "nan": "^2.11.1",
```
Comment (felixfbecker) [inline package.json]: This wasn’t a dependency before, but now it is. Seems unintentional?

File: package.json [hunk 2]
```diff
@@ -47,10 +47,12 @@
     "eslint-config-webpack": "^1.2.1",
     "eslint-plugin-import": "^2.2.0",
     "extract-text-webpack-plugin": "^3.0.0",
+    "fsevents": "^1.2.4",
```
Comment (felixfbecker) [inline package.json]: Why is this a dependency now? This should only be used on macOS

File: package.json (hunk 1 above)
Comment (kevinburke) [reply package.json]: Previously an older version of `nan` was present in package-lock.json - one that led to compile failures - and I wasn't sure how to get it to update besides declaring the version directly here.

File: package.json (hunk 2 above)
Comment (kevinburke) [reply package.json]: Previously an older version of `fsevents` was present in package-lock.json - one that led to compile failures - and I wasn't sure how to get it to update besides declaring the version directly here.

File: package.json (hunk 1 above)
Comment (felixfbecker) [reply package.json]: npm update should do that. If it doesn’t add it here, run npm install, then remove it here again, and run npm install again

**Review by felixfbecker — APPROVED** [review]

**Pull request merged**